server/service/src/login.rs

```diff
@@ -394,17 +395,20 @@ fn permissions_to_domain(permissions: Vec<Permissions>) -> HashSet<Permission> {
             // supplier invoices
             Permissions::ViewSupplierInvoices => {
                 output.insert(Permission::InboundShipmentQuery);
+                output.insert(Permission::OutboundReturnQuery);
             }
             Permissions::EditSupplierInvoices => {
                 output.insert(Permission::InboundShipmentMutate);
-                // TODO: WHICH?
-                output.insert(Permission::OutboundReturnMutate);
             }
+            Permissions::CreateSupplierInvoices => {
+                output.insert(Permission::InboundShipmentMutate);
+            }
+            // returns
             Permissions::ReturnStockFromSupplierInvoices => {
                 output.insert(Permission::OutboundReturnMutate);
             }
-            Permissions::CreateSupplierInvoices => {
-                output.insert(Permission::InboundShipmentMutate);
+            Permissions::ReturnStockFromCustomerInvoices => {
+                output.insert(Permission::InboundReturnMutate);
```

Comment:
Per the mSupply docs, these look like the right permissions to set to check if the user can create a return. 

However in OG it also looks like creating an outbound return (supplier credit) requires the 169 `CreateSupplierCredits` permission, and creating an inbound return (customer credit) requires the 42 `CreateCustomerInvoices` permission. 

I don't quite have my head around the distinction, and whether I should be enforcing both of those OG permissions be active here before inserting the mutate permission?

(Will follow up on this after Zen day 😇)

Reply:
Resolved, we'll just stick with the `ReturnStockFrom[X]Invoices` permissions for now 🚀

Reply:
Sounds good to me, would be great to have permission mapping doc somewhere at some point (so that when permissions are missing, either the support team or super user can reference vs looking at code), I'll ask on team lead chat what is the best medium of capturing these sorts of points

Reply:
https://github.com/msupply-foundation/msupply_docs/issues/24